### PR TITLE
Add created param example

### PR DESCRIPTION
**Pull request by ambarwootric**
Changes: Add created param example showing unix timestamp as requested by customer for clarification

Trello: https://trello.com/c/vPzxlYzr/6123-api-docs-to-show-an-example-post-with-real-createdle-parm

### CODE REVIEW

File: source/includes/_responses.md [hunk 1]
```diff
@@ -82,6 +82,43 @@ curl -H "Authorization: Bearer myaccesstoken" "https://api.wootric.com/v1/respon
     }
 ]
 ```
+```shell
+$TIMESTAMP = Time.utc(2016, 12, 25).to_i
```
Comment (hachibu) [inline source/includes/_responses.md]: I think you're mixing Ruby and Shell together in the same example.

Comment (hachibu) [reply source/includes/_responses.md]: For Bash, this might work...
```bash
 date -jf "%Y-%m-%d" "2016-12-25" "+%s"
```

Comment (ambarwootric) [reply source/includes/_responses.md]: Ah! makes total sense, thanks for pointing it out @hachibu. I have now updated it to `TIMESTAMP=$(date --date="2016-12-25" "+%s")` what do you think?

Comment (hachibu) [reply source/includes/_responses.md]: Looks good

**Review by hachibu — APPROVED** [review]
🚢

**Pull request merged**